cdap-tms/src/main/java/io/cdap/cdap/messaging/store/leveldb/LevelDBTableFactory.java

```diff
@@ -101,7 +101,7 @@ public LevelDBTableFactory(CConfiguration cConf) {
   @Override
   public void init() throws IOException {
     ensureDirExists(baseDir);
-    Path metaFile = Paths.get(baseDir.getAbsolutePath(), "meta");
+    Path metaFile = Paths.get(baseDir.getPath(), "meta");
```

Comment:
It is better to use the `Path.resolve` method instead.

`Path metaFile = baseDir.resolve("meta");

Reply:
Well, since `baseDir` is a file type, you can do 

`Path metaFile = new File(baseDir, "meta").toPath();` instead.

Reply:
Fixed